NitroxModel/DataStructures/GameLogic/InitialPingInstancePreferences.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NitroxModel.DataStructures.GameLogic
+{
+    [Serializable]
+    public class InitialPingInstancePreferences
+    {
+        public HashSet<string> HiddenSignalPings { get; set; }
+        public List<KeyValuePair<string, int>> ColorPreferences { get; set; }
```

Comment:
We discussed that Dictionary didn't worked initially. Did the provided fix not work? I would like to have a `Dictionary<>`, `IDictionary<>`, or `ThreadSafeDictionary` here more than `List<KeyValuePair<>>`

Reply:
it seems to work fine without the fix